README.md

```diff
@@ -1851,6 +1851,7 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 
 - [cState](https://github.com/cstate/cstate) - Static status page for hyperfast Hugo. Clean design, minimal JS, super light HTML/CSS, high customization, optional admin panel, read-only API, IE8+. Best used with Netlify, Docker. ([Demo](https://cstate.mnts.lt/)) `MIT` `Go`
 - [s.Status](https://github.com/scolastico-dev/s.Status) - Open-source server status page written in java. `MPL-2.0` `Java`
+- [StatPing.ng](https://statping-ng.github.io/) - An easy to use Status Page for your websites and applications. Statping will automatically fetch the application and render a beautiful status page with tons of features for you to build an even better status page. ([Source Code](https://github.com/statping-ng/statping-ng)) `GPL-3.0` `Docker`
```

Comment:
> not sure if you need Go installed

https://statping-ng.github.io/install.html -> https://raw.githubusercontent.com/statping-ng/statping-ng/stable/install.sh -> downloads and extracts the binary to `/usr/local/bin/`, it's a single ELF binary, there are no dependencies except a libc. In that case we use to include `Go` in the language tag since it's the way most Go programs are distributed (some use [go get](https://manpages.debian.org/bullseye-backports/gccgo-go/go.1.en.html))

-> `Docker/Go`